### ISSUE TITLE
Feature: return more info on getMe call.

### ISSUE THREAD

**Issue by sandro-salles** [open-wa/wa-automate-nodejs#483]
It would be nice to return more info on **getMe** call, specifically **isBusiness** and **isEnterprise** info.

Regards!

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i`);
exec(`npm run release-ci`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [1.9.47](https://github.com/open-wa/wa-automate-nodejs/compare/1.9.46...1.9.47) (2020-05-25)

- build(deps-dev): bump tsc-watch from 4.2.7 to 4.2.8 [`#484`](https://github.com/open-wa/wa-automate-nodejs/pull/484)
- Release 1.9.47 [`5af7c37`](https://github.com/open-wa/wa-automate-nodejs/commit/5af7c3791d5b9ab2715680198238033b4c6e0bd3)
- implemented safer `atob` [`24d751c`](https://github.com/open-wa/wa-automate-nodejs/commit/24d751c0201e47f3ade8dc3b40bf8afc7c4f3487)
- feature: more information in `getMe` #483 [`3ceaa38`](https://github.com/open-wa/wa-automate-nodejs/commit/3ceaa389a1b833fad241e33da603771e99521842)

**Comment by smashah** [comment]
@sandro-salles if `isBusiness` or `isEnterprise` is not true, it will return as undefined and is removed by the parser. So basically it will either be true or not a property at all.
